ui/component/notification/index.js

```diff
@@ -1,8 +1,14 @@
 import { connect } from 'react-redux';
```

Comment:
That's a known bug on master - we should not be sending that notification to already approved users.

Reply:
> That's a known bug on master - we should not be sending that notification to already approved users.

So that explains it 😅
![Peek 2021-08-18 14-37](https://user-images.githubusercontent.com/76502841/129969960-aef9f1f1-2e54-4037-bace-a79f76d43291.gif)